swarm/tools/validation/reporting/json_output.py

```
@@ -136,7 +136,7 @@ def build_detailed_json_output(
         flow_keys = get_flow_keys()
     except ImportError:
         # Fallback: use canonical 7-flow keys if registry not available
-        flow_keys = ["signal", "plan", "build", "review", "gate", "deploy", "wisdom"]
+        flow_keys = ["signal"] + ["plan"] + ["build"] + ["review"] + ["gate"] + ["deploy"] + ["wisdom"]
```

Ruff (0.15.2)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)